docs/source/en/model_doc/albert.mdx

```diff
@@ -67,22 +109,22 @@ This model was contributed by [lysandre](https://huggingface.co/lysandre). This
 
 ## AlbertModel
 
-[[autodoc]] AlbertModel
+[[autodoc]] AlbertModel 
```

Comment:
Remove the extra space at the end here. Same comment for all the other objects (`AlbertForPreTraining`, `AlbertForMaskedLM`, AlbertForSequenceClassification`, etc.) below.

```suggestion
[[autodoc]] AlbertModel
```